Gemfile.lock

```diff
@@ -213,4 +211,4 @@ DEPENDENCIES
   web-console (>= 3.3.0)
 
 BUNDLED WITH
-   2.0.1
+   2.1.4
```

Comment:
It is interesting that the Gem file updated. Hopefully it doesn't cause any problems!